views/scan.hbs

```diff
@@ -16,7 +16,7 @@
           <p>{{{fluentFormat req.supportedLocales "featured-breach-not-compromised" featuredBreach=featuredBreach.Title breachCount=foundBreaches.length}}}</p>
         {{/if}}
       {{else}}
-        <p>{{fluentFormat req.supportedLocales "scan-results" breachCount=foundBreaches.length}}</p>
+        <p>{{{fluentFormat req.supportedLocales "scan-results" breachCount=foundBreaches.length}}}</p>
```

Comment:
This made me file https://github.com/mozilla/blurts-server/issues/542 for follow-up, but doesn't need to block.